internal/proto/v2/common.proto

```diff
@@ -18,26 +18,40 @@ package s2a.proto.v2;
 
 option go_package = "github.com/google/s2a/internal/proto/v2/common_go_proto";
 
-// The ciphersuites for the TLS handshake supported by S2A. Note that handshake
-// ciphersuites are NOT configurable in TLS 1.3.
+// DEPRECATED: The ciphersuites for the TLS handshake supported by S2A. Note
+// that handshake ciphersuites are NOT configurable in TLS 1.3.
+// TODO: Delete this enum when it is no longer in use.
```

Comment:
Please assign all TODOs in the protos to my Github username.

Reply:
Done! I also reran the regenerate_proto.sh script to sync the protos and the generated code.